app/helpers/application_helper/button/cloud_volume_new.rb

```diff
@@ -13,7 +13,7 @@ def role_allows_feature?
   # disable button if no active providers support create action
   def disabled?
     ExtManagementSystem.none? do |ems|
-      "#{ems.class}::CloudVolume".safe_constantize.try(:supports_create?)
+      "#{ems.class}::CloudVolume".safe_constantize&.supports?(:create)
```

Comment:
may be time to fix the constantize :: mucking

```suggestion
      ::CloudVolume.class_by_ems(ems)&.supports?(:create)
```

Reply:
Agreed, think `ems.class_by_ems("CloudVolume")` is the preferred way.

We also have `ems.supports?(:cloud_volume_create)`